e2e-scripts/sippy-e2e-sippy-e2e-test-commands.sh

```
@@ -144,7 +131,63 @@ ${KUBECTL_CMD} -n sippy-e2e port-forward pod/redis1 ${SIPPY_REDIS_PORT}:6379 &
 
 ${KUBECTL_CMD} -n sippy-e2e get svc,ep
 
-${KUBECTL_CMD} -n sippy-e2e delete secret regcred
-
 # only 1 in parallel, some tests will clash if run at the same time
-gotestsum --junitfile ${ARTIFACT_DIR}/junit_e2e.xml -- ./test/e2e/... -v -p 1
+gotestsum --junitfile ${ARTIFACT_DIR}/junit_e2e.xml -- ./test/e2e/... -v -p 1 -coverprofile=${ARTIFACT_DIR}/e2e-test-coverage.out -coverpkg=./pkg/...,./cmd/...
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)